operator/data/profiles/demo.yaml

```diff
@@ -112,7 +112,7 @@ spec:
           targetPort: 15020
           name: status-port
         - port: 80
-          targetPort: 8080
+          targetPort: 80
           name: http2
         - port: 443
           targetPort: 8443
```

Comment:
These ones are wrong as well I think? It should be the same as the default profile except with the TCP port added I think?

Reply:
fixed. now matches with what's in `operator/data/profiles/default.yaml` with addition of TCP port. What tests are run against `demo` profile?  A little concerned this change was able to brake something as basic as request routing this close to release